src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/bolts/HaFlowCreateHubBolt.java

```diff
@@ -181,9 +183,11 @@ public void sendNotifyFlowMonitor(@NonNull CommandData flowCommand) {
 
     @Override
     public void sendPeriodicPingNotification(String haFlowId, boolean enabled) {
-        //TODO implement periodic pings https://github.com/telstra/open-kilda/issues/5153
-        log.info("Periodic pings are not implemented for ha-flow create operation yet. Skipping for the ha-flow {}",
-                haFlowId);
+        log.debug("Periodic ping ha-flow create operation. haFlowId={}", haFlowId);
+        PeriodicPingCommand payload = new PeriodicPingCommand(haFlowId, enabled);
```

Comment:
I thought it should be `PeriodicHaPingCommand`, not `PeriodicPingCommand`. How does it work?